pds_doi_service/core/util/config_parser.py

```diff
@@ -18,6 +18,16 @@
 
 class DOIConfigUtil:
 
+    @staticmethod
+    def _resolve_relative_path(parser):
+        # resolve relative path with sys.prefix base path
+        for section in parser.sections():
+            for (key, val) in parser.items(section):
+                if key.endswith('_file') or key.endswith('_dir'):
```

Comment:
According to the documentation you added, the path could already be specified as an absolute path. In that case we wouldn't want to `join` it with `sys.prefix`, so I think this condition also needs to check for `not os.path.isabs(val)`

Reply:
Thanks Scott, I checked that the os.path.join behaved as I expected for absolute path, and I think we are ok without handling the specific case of absolute paths. See:

os.path.join('/tmp/toto', '/home/loubrieu')
'/home/loubrieu'

Reply:
Oh neat, was not aware `join` had that behavior. Thanks for looking into it!